.github/workflows/build_and_test.yml

```diff
@@ -169,10 +169,10 @@ jobs:
         with:
           github_token: ${{secrets.GITHUB_TOKEN}}
           workflow: ci.yml
-          name: integritee-node-dev-61afcf67486a3ea3ec8d7f54feb7a064f6d94fc2
+          name: integritee-node-dev-8cfb2ff6623b3fa5aac182ac9c16293296e71a09
```

Comment:
This should be the merge into master : https://github.com/integritee-network/integritee-node/actions/runs/1663936560